jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -123,6 +123,11 @@
    */
   public static final String JAEGER_SAMPLER_MANAGER_HOST_PORT = JAEGER_PREFIX + "SAMPLER_MANAGER_HOST_PORT";
 
+  /**
+   * The config manager endpoint URL.
+   */
+  public static final String JAEGER_CONFIG_MANAGER_ENDPOINT = JAEGER_PREFIX + "CONFIG_MANAGER_ENDPOINT";
```

Comment:
We should use `CONFIG_MANAGER_HOST_PORT` as it is also used by baggage restriction. host port is also easier to configure.